pkg/apis/core/validation/validation.go

```diff
@@ -1644,7 +1647,8 @@ var allowedPVCTemplateObjectMetaFields = map[string]bool{
 // PersistentVolumeSpecValidationOptions contains the different settings for PeristentVolume validation
 type PersistentVolumeSpecValidationOptions struct {
 	// Allow spec to contain the "ReadWiteOncePod" access mode
-	AllowReadWriteOncePod bool
+	AllowReadWriteOncePod       bool
+	allowDNSSubDomainSecretName bool
```

Comment:
since this struct is an arg to a public method, this field should be public as well

Reply:
and add godoc for the field

Reply:
Done.. @liggitt .. thanks..